api/src/main/java/by/borge/jarl/Result.java

```diff
@@ -53,6 +54,28 @@ public <T> T getValue(Class<T> type) {
         ));
     }
 
+    /**
+     * Gets the value of this {@link Result} as a {@link List}.
+     *
+     * @param type the expected type of the returned list's elements
+     * @return this Result's value as a List
+     * @throws ClassCastException if the value isn't <code>null</code> or a {@link List}
+     */
+    public <T> List<T> getValueAsList(Class<T> type) {
+        List<?> list = getValue(List.class);
+        return list.stream().map(type::cast).collect(Collectors.toList());
+    }
+
+    /**
+     * Gets the value of this {@link Result} as a boolean.
+     *
+     * @return this Result's value as a boolean
+     * @throws ClassCastException if the value isn't <code>null</code> or a {@link Boolean}
+     */
+    public boolean getValueAsBoolean() {
+        return getValue(Boolean.class);
```

Comment:
Does this work? I thought we always returned a resultset / array thingy. A test would be good to assert this :)

Reply:
The (ResultSet)[https://github.com/borgeby/jarl/blob/main/api/src/main/java/by/borge/jarl/ResultSet.java] contains a list of `Result`.

Reply:
I see. It was the `getFirst` method that was missing from my equation. The test helped clarify that 👍